src/version.c

```diff
@@ -20,21 +20,39 @@
 #define STR_EXPAND(x) #x
 #define STR(x) STR_EXPAND(x)
 
-static const char *git_commit = STR(EXT_GIT_COMMIT);
+static const char commit_sha[] = EXT_GIT_COMMIT_SHA;
```

Comment:
Do we want to make this an exported string, e.g., `ts_commit_sha` so that one can use `strings` and `grep` directly on the binary?

Reply:
I was considering this, but we might want to have some sort of tag so that we can easily find it, e.g., `sha:aec070645fe53ee3b3763059376134f058cc337247c978add178b6ccdfb0019f` and the same for the tag.

Reply:
I created three tags `commit-sha:`, `commit-tag:`, and `commit-time:` and added them. I do not think we need to export them since they are visible in the SO anyway as long as the link-time optimizer does not remove them. The prefix is removed before added to the result.